filter-queries/src/main/java/com/cognitivescale/services/CortexManagedContent.java

```diff
@@ -0,0 +1,55 @@
+package com.cognitivescale.services;
+
+import com.cognitivescale.models.CortexContentTag;
+import com.cognitivescale.models.CortexResponse;
+import com.cognitivescale.utils.CortexHelper;
+import com.google.gson.Gson;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+public class CortexManagedContent extends CortexHelper implements ManagedContent {
+
+    private static final String uriFormat = "/fabric/v%d/projects/%s/content/%s";
+
```

Comment:
Maybe, haven't really thought about it. I'd rather not have this kind of integration, as we have other tools that provide a client for managed content. Then again it might be useful just to pull it in.